### ISSUE TITLE
[Bug]: Login was not successful!

### ISSUE THREAD

**Issue by mawoIT** [homebridge-eufy-security/plugin#615]
### What happened?

A bug happened!
Hello, after the last update I can no longer log into the Eufy Cloud in the plugin. The error message is: Login was not successful!
I have already tried it with several users.

### Device Type

Station (Supported)

### Plugin Version

v3.0.x (Best Effort)

### HomeBridge Version

v1.7.x (Supported)

### NodeJS Version

v20 (Supported)

### Operating System

Other (UnSupported)

### Relevant log output

```shell
Login was not successful!
```

### config

```shell
{
    "platform": "EufySecurity",
    "name": "EufySecurity",
    "username": "homebridge@*********",
    "password": "*********",
    "country": "AT",
    "deviceName": "",
    "ignoreStations": [],
    "stations": [
        {
            "hkHome": 1,
            "hkAway": 0,
            "serialNumber": "*********",
            "hkNight": 3
        }
    ],
    "ignoreDevices": [
        "*********",
        "*********"
    ],
    "cameras": [
        {
            "serialNumber": "*********",
            "rtsp": false
        }
    ]
}
```

**Comment by OrigSorceror** [comment]
Yes I too am getting the same result.  I am also unable to log in the the mysecurity.eufylife.com website. I dare say you are experiencing a similar issue.  Also are unable to login via the eufy app on my phone unless i leave my home network and use Cellular then it works fine.  Camera streams not being blocked. I can view then via the eufy app on phone whilst on cellular network.  But everything on the Eufy plugin drops out of the plugin as the plugin can not log in to its user account. so plugin no longer sees the Cameras, the homebridge, or motion sensors. 

Not sure what on my network could be blocking it.  I am thinking the ISP is likely blocking it as it is detecting all the push notifications from my 8 motion sensors as DDOS attacks maybe.  so if puta block on the login to eufylife via their web page on a PC or via the app.. 

after a few hours it fixes itself (or maybe i think it does as i usually uninstall the plugin and re-install it. a real PITA.   Not really sure how to reslove this issue.  I doubt it is anything in my network as nothing has changed in my network for months. I even dropped my firewall and still fails to allow me to login to the console from within home network. 

Might have to try running through a VPN service to get around the ISP blocks if that is indeed happening.

**Comment by OrigSorceror** [comment]
Confirmed it is the ISP blocking the Access to the eufy consols.  I enable Private internet Access on my PC and can log in to the console fine..  turn off PIA  and back to Network Error.

**Comment by lenoxys** [comment]
I've 6 cameras doesn't make so mush noise on my side. I don't know why ISP or Eufy is blocking your ISP.

**Comment by OrigSorceror** [comment]
Yeah i have 4 cameras on the outside of the house and 4 eufy motion detectors inside to turn on lights at night time. But the are constantly sending motion detected=true and motion detected=false packets to homebridge 24x7. ISP may detect thes 1000’s of packets each day as DOS attacks. And temp ban the site.  Only explanation i have of why it stops working then starts working few hours later. Oh and i restart homebridge as well and when the log in is blocked it removes all eufy devices from homekit and diables the lighting automation i have set up.

**Comment by nanoupi42** [comment]
Hi,
Exactly the same problem for me :
I can't log on Eufy App if I'm using Wifi, and same problem on the website :
https://mysecurity.eufylife.com/#/login?type=/camera

I have tried to access this website with another IP and It's working
When I tried from my ISP with this IP I have a message : Network Error

I have routed the traffic through a VPN with another IP  to connect the eufy app again and it's working !
If I reboot the eufy container, the cameras are gone...

**Comment by OrigSorceror** [comment]
yes, not an issue with the plugin perse'.  It is either Eufy has changed their security settings, or it is an ISP issue. Not sure which at this stage. Routing everything thru a VPN tends to resolve the issue.  But it is cumbersome, as it adds increased latency.

**Comment by hrotsvith** [comment]
However, 3.4.2 still works flawlessly, 4.x not. So from my view, something has changed in the login process in 4.x which causes the problem

**Comment by OrigSorceror** [comment]
> However, 3.4.2 still works flawlessly, 4.x not. So from my view, something has changed in the login process in 4.x which causes the problem

Interesting.  Might try rolling the plugin back then.  Wonder if the app is causing too many login attempts and Eufy blocks the IP address temporarily.

**Comment by OrigSorceror** [comment]
Darn, rolling it back whilst blocked  :(  It removed all my devices again :(..  Have to wait now until Eufy/ISP unblock to test the Rollback to 3.4.2

**Comment by OrigSorceror** [comment]
Ok Eufy has let me back in :)  and the devices have re-registered using the 3.4.2 plugin version.. Will see how this goes :)

**Comment by mariomaz87** [comment]
Same issue here. I live in Italy and my provider is Fastweb, I guess you have different providers so it's not something related to the ISP.

**Comment by lenoxys** [comment]
> Ok Eufy has let me back in :)  and the devices have re-registered using the 3.4.2 plugin version.. Will see how this goes :)
> 
> 

Let me know. I think I know why

**Comment by OrigSorceror** [comment]
> > Ok Eufy has let me back in :)  and the devices have re-registered using the 3.4.2 plugin version.. Will see how this goes :)
> 
> Let me know. I think I know why

Been rock solid since i reverted to 3.4.2... :) 
only error i am getting since i reverted is. 
(node:13685) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 12 shutdown listeners added to [HomebridgeAPI]. Use emitter.setMaxListeners() to increase limit
(Use `node --trace-warnings ...` to show where the warning was created)

other than that no issues at all.

**Comment by lenoxys** [comment]
duplicate of #608
